service/misc_services.cc

```diff
@@ -277,9 +277,6 @@ future<> view_update_backlog_broker::on_change(gms::inet_address endpoint, const
         char* end_bound;
         for (auto* ptr : {&current, &max}) {
             *ptr = std::strtoull(start_bound, &end_bound, 10);
-            if (*ptr == ULLONG_MAX) {
-                return make_ready_future();;
-            }
```

Comment:
As I wrote under the discussion on the issue, I think the problem you encountered was due to incorrect error handling which got triggered on empty backlog. I think a better direction would be to fix the error handling logic - instead of looking at the returned value, look at the `errno`.

Reply:
Ofc, it would also make sense to do it for the `ticks == LLONG_MAX` check.

Reply:
I added the errno checks and updated the commit message